src/utilities/theme.ts

```diff
@@ -21,6 +21,20 @@ const theme = createTheme({
       },
     },
   },
+
+  typography: {
+    h2: {
+      fontFamily: 'Rubik',
+      fontSize: '20px',
+      fontWeight: '600',
+    },
+
+    body1: {
+      fontFamily: 'Inter',
+      fontSize: '16px',
+      fontWeight: '300',
```

Comment:
No protótipo o peso de fonte de body é 400.

```suggestion
      fontWeight: '400',
```

Reply:
qual é a diferença entre texto e corpo? um tá 400 e outro 300

![image](https://user-images.githubusercontent.com/44441530/184453834-c4b7bc15-f97a-410b-8b58-2e95a2e994bd.png)

Reply:
texto seriam todos os textos que informam o usuário sobre algo, já o corpo seriam textos como: código de matéria, ano/semestre, e afins.

Reply:
No nosso protótipo está corretamente empregado os dois pesos.